src/__configuration__/navigationMenu/navigation.tsx

```diff
@@ -373,6 +373,12 @@ export const pageDefinitions: SimpleNavItem[] = [
         hidden: true,
         component: <MarkdownPage title={'Brightlayer UI Rebrand'} markdown={Docs.Community.Rebrand} />,
     },
+    {
+        title: 'Brightlayer UI Package Migration',
+        url: '/brightlayer-ui-package-migration',
+        hidden: true,
+        component: <MarkdownPage title={'Brightlayer UI Package Migration'} markdown={Docs.Community.Migration} />,
+    },
```

Comment:
Why is it a hidden page?

Reply:
Not sure... I copied from other routes :grimacing:.

Reply:
@joebochill we want this page hidden from drawer & global search? Or visible under release notes? Here's my thought but totally open to new suggestions. 

```suggestion
    {
        title: 'Migrate from PX Blue Packages',
        url: '/brightlayer-ui-package-migration',
        component: <MarkdownPage title={'Brightlayer UI Package Migration'} markdown={Docs.Community.Migration} />,
    },
```

Reply:
If I remove hidden: true, it shows in drawer menu, but not showing page with search text `migration`.
<img width="948" alt="Screenshot 2021-12-20 at 6 23 42 PM" src="https://user-images.githubusercontent.com/10433274/146770288-dc1d9429-7216-4184-8d0f-14b15b32064a.png">

I am fine with above suggestions. @emclaug2 any thoughts ?

Reply:
>  it shows in drawer menu, but not showing page with search text migration.

Did you run `yarn indexer`?

Reply:
Yes, still not showing.